### PR TITLE
Add export to Excel functionality

### LINKED ISSUE #632: [FEAT] On "Export to Excel," create and download .xls file of relevant chart data
# ⚠️ IMPORTANT: Please fill out this template to give us as much information as possible to consider/implement the feature.

### Prerequisites

- [x] check this box if you have completed the following:
    - Reviewed the [contributing guidelines](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md) and [support](https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md) files
    - Reviewed the README file for the repository you are working in
    - Searched for relevant instructions [on our Discord server](https://discord.gg/pPERUuv)
    - Searched the issues of the repository you are working in to make sure one was not already filed

### Summary

- [ ] On click of "Export to Excel," use a third-party package such as [react-export-excel](https://www.npmjs.com/package/react-export-excel) to translate all displayed information in the current "Show Me More" page into an Excel file
- [ ] Ensure .xls file is downloaded 

### Motivation

This is one of the most-requested features for our full launch

### PR DESCRIPTION
**Pull request by SomeMoosery**
# ⚠️ __IMPORTANT: Please do not create a Pull Request without creating an issue first.__

All changes need to be discusssed before proceeding. Failure to do so may result in the pull request being rejected.

Please be sure to review our [Code of Conduct](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CODE_OF_CONDUCT.md), [Contributing Guidelines](https://github.com/COVID-19-electronic-health-system/.github/blob/master/CONTRIBUTING.md), and [Support documentation](https://github.com/COVID-19-electronic-health-system/.github/blob/master/SUPPORT.md) before submitting a pull request.

-----

## Please include the issue number the pull request fixes by replacing YOUR-ISSUE-HERE in the text below.

Fixes #632 

## Summary

- Downloads observations as an Excel file on button on "Show Me More"

**Currently still co-authoring with @rachelafrick**

## Details

Screen Shot 2020-05-07 at 10 44 18 AM

## Final Checklist

- [x] For CoronaTracker, did you bump the version in `package.json`?
    - Update the second decimal for a major change
    - Update the third decimal for a minor change
    - Numbers can go past 9, e.g. `1.0.9` => `1.0.10`
    - For more info, read about [Semantic Versioning](https://semver.org/)
- [x] Did you add any new tests as necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits? _(can be done at merge)_
- [x] Did you use `yarn` not `npm`? _(important!)_
- [x] Did you use Material-UI wherever possible?
- [x] Did you run `yarn lint` on the code?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: client/src/components/showMeMore/Download.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  export: { ...buttonsCss.buttons, margin: theme.spacing(1) },
+}));
+
+const Download = props => {
+  const { observations } = props;
+  const classes = useStyles();
+
+  // Add all of our current observations to the data set
+  useEffect(() => {
+    observations.map(observation => {
```
Comment (acthelemann) [inline client/src/components/showMeMore/Download.jsx:44]: Let's use `forEach` instead

File: client/src/components/showMeMore/Download.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        observation.physical.chestPainSeverity,
+        observation.physical.fatigueSeverity,
+        observation.physical.soreThroatSeverity,
+        observation.physical.bluishnessSeverity,
+        observation.nonPhysical.openComment,
+      ];
+      dataSet[0].data.push(temp);
+      return temp;
+    });
+  });
```
Comment (acthelemann) [inline client/src/components/showMeMore/Download.jsx:63]: I think we should add `observations` to the dependency array

File: client/src/components/showMeMore/Download.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import PropTypes from 'prop-types';
+import buttonsCss from '../../css/buttons';
+
+const { ExcelFile } = ReactExport;
+const { ExcelSheet } = ReactExport.ExcelFile;
+
+// Pre-defined dataset for export
+const dataSet = [
+  {
+    columns: [
```
Comment (acthelemann) [inline client/src/components/showMeMore/Download.jsx:15]: `chestPainSeverity` is no longer used

Missing physical: `giIssueSeverity`, `headacheSeverity`, `lostSmellSeverity`, `lostTasteSeverity`

Do we want to include emotional stuff? `appetite`, `energy`, `interest`, `sadness`, `sleep`?

Ideally, we have all of these fields stored in a constants file or something for less hard coding

Comment (SomeMoosery) [reply client/src/components/showMeMore/Download.jsx]: Added emotional stuff and moved `dataSet` into constants

File: client/src/components/showMeMore/Download.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        observation.physical.lostTasteSeverity,
+        observation.nonPhysical.appetite,
+        observation.nonPhysical.energy,
+        observation.nonPhysical.interest,
+        observation.nonPhysical.sadness,
+        observation.nonPhysical.sleep,
+        observation.nonPhysical.openComment,
+      ];
+      dataSet[0].data.push(temp);
+      return temp;
```
Comment (acthelemann) [inline client/src/components/showMeMore/Download.jsx:48]:
```suggestion
```

It's not going to break anything, but the `forEach` callback doesn't return anything

File: client/src/utils/constants.js [hunk 5]
```diff
@@ -238,4 +238,33 @@ export const states = [
...
+      'Daily Symptoms Feeling',
+      'Daily Compared to Yesterday',
+      'Fever Severity',
+      'Shortness of Breath Severity',
+      'Chills Severity',
+      'Cough Severity',
+      'Fatigue Severity',
+      'Sore Throat Severity',
+      'Bluishness Severity',
+      'GI Issues Severity',
```
Comment (acthelemann) [inline client/src/utils/constants.js:255]:
```suggestion
      'Gastrointestinnal Issues Severity',
```

If you think it needs to be more readable

**Pull request merged**